frontend/src/shared/model/app.configuration.model.ts

```diff
@@ -114,17 +114,17 @@ export const defaultConfiguration: IAppConfiguration = {
             transformJob: []
         },
         EnabledServices: {
-            batchTranslate: false,
-            realtimeTranslate: false,
-            emrCluster: false,
-            endpoint: false,
-            endpointConfig: false,
-            hpoJob: false,
-            labelingJob: false,
-            model: false,
-            notebook: false,
-            trainingJob: false,
-            transformJob: false,
+            batchTranslate: true,
```

Comment:
Revert this; we purposely want the default app config to be very restrictive - this is to account for a situation where the app config API couldn't be reached and we want to err on the side of caution.